community/kernel/src/main/java/org/neo4j/kernel/impl/store/record/IndexRule.java

```diff
@@ -114,6 +114,11 @@ public Long getOwningConstraint()
         return owningConstraint;
     }
 
+    public boolean isNotValidatedIndexConstraint()
```

Comment:
As with all boolean methods it's way nicer if they are positive in nature... easier to grok when read in conditions.